modules/plugins/utility/motion/precognition/default.nix

```diff
@@ -0,0 +1,6 @@
+_: {
```

Comment:
```suggestion
{
```

Function argument can be omitted if empty.

Reply:
Resolved in b0641a8c0b4e1384af0c2017ca659883befa93d2.